docs/building-web.md

```diff
@@ -13,8 +13,8 @@ This document describes how to build and test Accessibility Insights for Web (th
 
 You will need the following tools installed:
 
--   [Node](https://nodejs.org) >= 12.16.3 (check by running `node --version`) - This is the version being enforced on our builds
--   [Yarn](https://yarnpkg.com/getting-started/install) >= 1.22.4 (check by running `yarn --version`)
+-   [Node](https://nodejs.org) >= 14.15.0 (check by running `node --version`) - This is the version being enforced on our builds
```

Comment:
Should we defer/adjust this change to the instructions until we update `package.json`? We can still build with 12.16.3.

Reply:
I wanted to encourage the use of the same version CI is building against, less room for error that way